.vib/phpmyadmin/cypress/cypress.json

```diff
@@ -1,8 +1,9 @@
 {
-    "baseUrl": "http://localhost/",
-    "env": {
-      "username": "root",
-      "password": "rootPassword"
-    },
-    "defaultCommandTimeout":30000
-  }
+  "baseUrl": "http://localhost/",
+  "env": {
+    "username": "root",
+    "password": "rootPassword"
+  },
+  "defaultCommandTimeout": 30000,
+  "pageLoadTimeout": 240000
```

Comment:
Is this something we should be adding by default from now on?

Reply:
As this is something depending on the application, I'd prefer to have the developer estimate the need.
In the case of flakiness prone apps (Grafana, this one) this is needed. 
For others (Dokuwiki, Mediawiki) there is no need since the app performs well in the flakiness test.